### DISCUSSION TITLE
Fedora 36 "SteamVR not installed: make sure you launched it at least once, then close it."

### DISCUSSION THREAD

**Discussion by tekchip** [alvr-org/ALVR#1104] [Q&A]
I've tried running the regular package of ALVR and the portable package as well. Both run but display the message:

"SteamVR not installed: make sure you launched it at least once, then close it."

I'm running Steam Flatpak (perhaps this is the problem?) on Fedora 36 and I have installed and run Steam VR several times at this point. SteamVR seems to run fine on it's own.

The reported path that Steam VR is installed to is /home/tekchip/.var/app/com.valvesoftware.Steam/.local/share/Steam/steamapps/common/SteamVR I seem to recall having seen Steam install's stuff to .Steam instead of .var on other distros. I'm honestly not sure why this is different. To my knowledge I didn't change this. So perhaps this is the problem?

Also of note the localhost:8082 page does not seem to load/work.

I thought maybe some security preventing it from loading properly, so I tried running it via sudo. I know, ill advised probably, but this made no difference.

Running from the CLI doesn't show any useful information or errors.

`bin]$ ./alvr_launcher
DEBUG [druid::localization] available locales [], current en-US
DEBUG [druid::localization] resolved: [en-US]
INFO  [druid_shell::platform::gtk::application] gtk: Activated application`

**Comment by ColdIce1605** [comment]
flatpak isn't really supported some people have gotten some steps on getting it working but for now only use the non-flatpak version on your distro.
